src/sparseml/pytorch/utils/exporter.py

```diff
@@ -305,10 +305,10 @@ def export_samples(
         :param exp_counter: the counter to start exporting the tensor files at
         """
         sample_batches = [tensors_to_device(batch, "cpu") for batch in sample_batches]
-        inputs_dir = os.path.join(self._output_dir, "sample-inputs")
-        outputs_dir = os.path.join(self._output_dir, "sample-outputs")
-        labels_dir = os.path.join(self._output_dir, "sample-labels")
-        originals_dir = os.path.join(self._output_dir, "sample-originals")
+        inputs_dir = os.path.join(self._output_dir, "sample_inputs")
```

Comment:
QQ: Do we rely on the original naming conventions anywhere? If we do it would be nice to have a quick test